dist/rei-dot-com/scss/cdr-tokens.scss

```diff
@@ -2284,6 +2284,144 @@ $cdr-membership-subtle-color-border-button-primary-rest: #13352c;
 $cdr-membership-subtle-color-border-button-primary-active: #13352c;
 $cdr-membership-subtle-color-border-button-primary-hover: #13352c;
 
+// @supports statements can be removed once Safari 15 is no longer supported
+
+// XS
+@mixin cdr-xs-cq-up($name: '') {
+  @container #{$name} (min-width: #{$cdr-breakpoint-xs}) {
+    @content;
+  }
+
+  @supports not (contain: inline-size) {
+    @media (min-width: #{$cdr-breakpoint-xs}) {
+      @content;
+    }
+  }
+}
+
+@mixin cdr-xs-cq-only($name: '') {
+  @container #{$name} (min-width: #{$cdr-breakpoint-xs}) and (max-width: calc(#{$cdr-breakpoint-sm} - 1px)) {
+    @content;
+  }
+
+  @supports not (contain: inline-size) {
+    @media (min-width: #{$cdr-breakpoint-xs}) and (max-width: calc(#{$cdr-breakpoint-sm} - 1px)) {
+      @content;
+    }
+  }
+}
+
+// SM
+@mixin cdr-sm-cq-down($name: '') {
+  @container #{$name} (max-width: calc(#{$cdr-breakpoint-sm} - 1px)) {
+    @content;
+  }
+
+  @supports not (contain: inline-size) {
+    @media (max-width: calc(#{$cdr-breakpoint-sm} - 1px)) {
+      @content;
+    }
+  }
+}
+
+@mixin cdr-sm-cq-up($name: '') {
+  @container #{$name} (min-width: #{$cdr-breakpoint-sm}) {
```

Comment:
Another question for you 
are the min and max fixed values here useful? 
This seems to mix legacy media query ideas with future more fluid ideas? 
is it accomplishing more than say

@mixin fluid-container($name: '') {
  @container #{$name} (width >= 50vw) {
    @content;
  }
}

I'd be interested in something like this 


  --cdr-container-size-xxs: '';
   --cdr-container-size-xxl: '';
   
   @mixin cdr-fluid-container($breakpoint: var(--container-breakpoint), $name: '') {
  @container #{$name} (width >= $breakpoint) {
    @content;
  }
}

Reply:
Hmm, I do think it helps to have a set of known options for designers/devs to choose from for container query breakpoints—I'm thinking of pixel values when I say this. Having a set of breakpoints gives a set of options that designers are sort-of forced into, which can be helpful.

For that example mixin you mentioned, my concern would be that devs would need to manually specify the breakpoint—they can do this already, so it's not adding a lot of value. It speeds dev and designer time up by already having a set of these breakpoint options....maybe this means giving more options than just xs/sm/md/lg, but I think the concept holds true. Lets chat about this one too